### ISSUE TITLE
Add accessibility tests for Control and Input components

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#4284]
Accessibility tests should be added for the following components:

**Controls:**

- [x] Accordion
- [x] Anchor
- [x] Drop
- [x] DropButton
- [x] Menu
- [x] Nav
- [x] Tabs

**Input:**

- [x] CheckBox
- [x] MaskedInput
- [x] RadioButton
- [x] RadioButtonGroup
- [x] RangeInput
- [x] RangeSelector
- [x] TextInput
- [x] FormField

**Comment by ShimiSun** [comment]
EXCELLENT work!!
